components/descriptions/Cell.tsx

```diff
@@ -63,7 +63,11 @@ const Cell: React.FC<CellProps> = ({
           {label}
         </span>
       )}
-      {content && <span className={classNames(`${itemPrefixCls}-item-content`)}>{content}</span>}
+      {content && (
+        <span className={classNames(`${itemPrefixCls}-item-content`)}>
+          <div>{content}</div>
```

Comment:
I don't think put a div into span is a good idea.

Reply:
Maybe we should consider other element?

Reply:
@Rustin-Liu Thanks for the comment.

I agree. I found a related issue on SO:

https://stackoverflow.com/questions/43690276/text-selection-inside-inline-blocks-without-space-between

Reply:
@Rustin-Liu What do you think?

Reply:
It's OK for me. Thanks!